### PR TITLE
Home work1

### PR DESCRIPTION
**Pull request by Eidoline**

### CODE REVIEW

File: AutomationCourse_HomeWork/Task_1.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace AutomationCourse_HomeWork
+{
+    class Task_1
+    {
+        static void Main()
+        {
+            Console.WriteLine("Please add three arguments for the futher operations:");
+            Console.Write("x = ");
+            double x = Convert.ToInt32(Console.ReadLine());
```
Comment (NaGriS) [inline AutomationCourse_HomeWork/Task_1.cs]: variable got type double, but we are doing cast to int, why?

**Review by NaGriS — APPROVED** [review]
Without checking that user input is correct - all is great. Amendments to the howework we will consider later

File: AutomationCourse_HomeWork/Task_1.cs (hunk 1 above)
Comment (Eidoline) [reply AutomationCourse_HomeWork/Task_1.cs]: Fixed.

**Comment by Eidoline** [comment]
>

**Comment by Eidoline** [comment]
There were no information about user input at all, also we have not learned if-else constructions yet, so there are no any checks for data to simplify programs assuming user being aware of tasks.

**Pull request merged**